spec/controllers/authorizations_controller_spec.rb

```diff
@@ -154,7 +154,7 @@ def translated_error_message(key)
 
     it 'should redirect immediately' do
       expect(response).to be_redirect
-      expect(response.location).to match(/oauth\/authorize\/native\?code=#{Doorkeeper::AccessGrant.first.token}/)
+      expect(response.location).to match(/oauth\/authorize\//)
```

Comment:
Do we need to assert that the code is present? Here and below https://github.com/procore/doorkeeper/pull/1/files#diff-fd7a37fac4561d29fdcd1a31055ceddfL32

Reply:
I think we could, this is just simply backing out the changes that doorkeeper had made when they changed the api. but I'm happy with more testing!